components/scream/src/physics/p3/atmosphere_microphysics.hpp

```diff
@@ -246,9 +247,11 @@ class P3Microphysics : public AtmosphereProcess
         diag_eff_radius_qc(icol,ipack) *= 1e6;
         diag_eff_radius_qi(icol,ipack) *= 1e6;
       } // for ipack
+      precip_liq_surf(icol) += precip_liq_surf_local(icol) * PC::RHO_H2O * m_dt;
```

Comment:
I'd rather _local was _flux to clarify what it means...

Reply:
So do I, but there is already a variable in P3 and in the memory buffer of the interface called `precip_liq/ice_flux`.  I didn't want to go down the rabbit hole of whether or not that variable could be (a) changed in name or (b) used instead of the precip_liq/ice_surf that P3 creates.  I suspect the latter.  The _flux versions are just 3D variables that record the precipitation at every layer, including the surface.

Reply:
Uh oh. It doesn't bug me that we already have `precip_liq_flux` in the FM since `precip_liq_**surf**_flux` is a different variable. Optimal would be to name `precip_liq_flux`=> `precip_liq_**3d**_flux` to disambiguate, but maybe that's too much to ask. 

The bigger problem is that *all* fluxes should be averaged over their substeps and it sounds like we missed this one (well, two since there's also `precip_ice_flux`). Is it actually used for anything other than output?

Reply:
TBH I didn't dig too deep to see what it is used for.  I can make surf_flux the naming convention.

Reply:
👍